README.md

```diff
@@ -223,6 +225,7 @@ The supported categories are described below:
 #### Rules
 
 Each category contains a group of rules. The spec that each rule applies to is marked in the third column. For the actual configuration structure, see the [default values](#default-values).
+You can use the `-v` option of the CLI validator to display the rule for each reported result.
```

Comment:
This may not be the place for it but I think it may be helpful to mention in the README somewhere that messages with rule "builtin" cannot be customized.